src/arrays.js

```diff
@@ -1,62 +1,157 @@
 const getNthElement = (index, array) => {
-  // your code here
+   //pass
+  /*We need a more dynamic way including modulus
+   because using -4 is not dynamic meaning we would 
+   have to keep making new code for any updates!!!
+   .length is what we need to use but I have tried 
+   and not figured out the correct method yet! */
+   while (index >3) {index = index -4};
+   return array[index];
+   /*but this works for now*/
+    
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  //pass
+  return array.toString('')
 };
 
 const csvStringToArray = string => {
-  // your code here
+  //pass
+  return string.split(",")
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  // pass
+  array.push(element);
+  //array.push([element]);
+  //array.concat(element);
+  //array.concat([element]);
+  //array.push(4)
+  //array.concat(4)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  
+ return array.concat(element);
+//array.push(element);
+ //return array
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+    return array.splice(index,1);
+   
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  //pass
+  /*numbers.toString converts the entire array to a string, .split(",")
+   let me change each individual number to a string */
+  return numbers.toString().split(",")
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  //pass
+  /* strings.map lets me select all the strings in the array and create a new array,
+  the array method .map to .toUpperCase changes the array to uppercase*/
+  return strings.map((strings => strings.toUpperCase()))
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+ //pass
+  /*return strings.reverse = reverses the order of the array
+ The split() method splits a String object into an array of string 
+ by separating the string into sub strings.
+The reverse() method reverses an array in place. The first array 
+element becomes the last and the last becomes the first.
+The join() method joins all elements of an array into a string.
+last I added .map, .map selects each individual element in array 
+(I struggled to figure how to select the individual elements .map was the one)*/
+ return strings.map(strings => strings.split("").reverse().join(""));
 };
 
 const onlyEven = numbers => {
-  // your code here
+  //pass
+  /*filter function maybe??? 
+  using filter and then giving a function named that function X.
+  X % 2 should give back all even numbers but I don't quite understand the maths per usual
```

Comment:
maths is simple here

`% 2` is saying

divide the number by 2, if the remainder is zero, then you have an even number :)